### PR TITLE
Add point to direction block

### PR DESCRIPTION
**Pull request by nkiruka**
This PR adds the point to direction block to artist.

screen shot 2018-03-22 at 5 48 31 pm

### CODE REVIEW

**Comment by joshlory** [comment]
Is implementing `ArtistAPI.prototype.pointTo` part of a follow-on PR?

**Comment by nkiruka** [comment]
Yes, implementing ArtistAPI is part of the next PR

File: apps/src/turtle/blocks.js [hunk 1]
```diff
@@ -259,6 +259,28 @@ exports.install = function (blockly, blockInstallOptions) {
     }
   };
 
+blockly.Blocks.point_to = {
+    // Block for pointing to a specified direction
+    helpUrl: '',
+    init: function () {
+      this.setHSV(184, 1.00, 0.74);
+      this.appendDummyInput()
+      .appendTitle(msg.pointTo());
```
Comment (balderdash) [inline apps/src/turtle/blocks.js]: nit: indent this line like you did below

Comment (nkiruka) [reply apps/src/turtle/blocks.js]: Fixed!  Thanks

**Pull request merged**